tests/sanity/fake_sanity_mounter.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2024 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sanity
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/cloud"
+	"k8s.io/mount-utils"
+)
+
+type fakeMounter struct {
+	mounts map[string]string
+}
+
+func NewFakeMounter() *fakeMounter {
```

Comment:
non-blocking: if we don't need it outside of the package, we can probably make it lowercase e.g., `newFakeMounter`.

Reply:
Done in new rev thank you !